### PR TITLE
Validate checbox not required

### PR DESCRIPTION
**Pull request by barchen1**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/23601

## Description
Added  a validation that checkbox items are not required in integrations

## Screenshots

Screen Shot 2020-05-19 at 17 26 00

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 1]
```diff
@@ -148,6 +150,40 @@ def is_insecure_configured_correctly(self):
...
+        # type: () -> bool
+        """Check that if an integration has a checkbox parameter it is configured properly.
+        Returns:
+            bool. True if the checkbox parameter is configured correctly, False otherwise.
+        """
+        err_msgs = []
+        configuration = self.current_file.get('configuration', [])
+        for configuration_param in configuration:
+            param_name = configuration_param['name']
+            if configuration_param['type'] == 8 and param_name not in ('insecure', 'unsecure', 'proxy', 'isFetch'):
```
Comment (ronykoz) [inline demisto_sdk/commands/common/hook_validations/integration.py:163]: no need to check the param name, all checkboxes cannot be required

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 2]
```diff
@@ -148,6 +150,40 @@ def is_insecure_configured_correctly(self):
...
+                err_msg = self.is_valid_checkbox_param(configuration_param, param_name)
+                if err_msg:
+                    err_msgs.append(err_msg)
```
Comment (ronykoz) [inline demisto_sdk/commands/common/hook_validations/integration.py:166]: let's not aggregate, let's just print them out - can be done in the other funciton

Comment (barchen1) [reply demisto_sdk/commands/common/hook_validations/integration.py]: done

File: demisto_sdk/commands/common/hook_validations/integration.py [hunk 3]
```diff
@@ -148,6 +150,35 @@ def is_insecure_configured_correctly(self):
...
+        # type: (dict, str) -> None
+        """Check if the given checkbox parameter required field is False.
+        Returns:
+            str. Empty string if valid, error message otherwise.
+        """
+        err_msg = None
+        if configuration_param.get('required', False):
+            err_msg = Errors.wrong_required_value(param_name)
+        if err_msg:
+            self.is_valid = False
```
Comment (ronykoz) [inline demisto_sdk/commands/common/hook_validations/integration.py:179]: better to return true false then to update this param

Comment (barchen1) [reply demisto_sdk/commands/common/hook_validations/integration.py]: done

**Pull request merged**